sig-operations/charter.md

```diff
@@ -0,0 +1,37 @@
+# SIG Operations
+
+## Scope
+
+The goal of this SIG is to define and coordinate management and operations for hardware and platform tooling within the
+Operate First Community Cloud (op1st)
+
+### In scope
+
+* Define what is part and not part of the Operate First Cloud platform offering
+* Communicate a platform roadmap
+* Day to day operations of platform
+* Ensuring the stability of the platform
+* Upgrading the platform
+* Coordinate hardware and infrastructure provisioning
+* Resolving platform level incidents
+* Providing GitOps framework for managing platform
+* Providing Automation Runbooks for self servicing repetitive operations tasks
+* PR / Code reviews pertaining to cluster/platform management
+* Documentation of all above
+* Ensure recording decisions via the ADR process
+* Ensure inclusivity and transparency
+
+### Out of scope
+
+* User workload operations
+* Procuring hardware and infrastructure
+
+## Stakeholders
+
+* User workload operations
+* Procuring hardware and infrastructure
```

Comment:
This looks like a copy paste error to me. These are not stakeholders. Also, please correct me if I'm wrong, I don't think SIG requires stakeholders, because SIG is the stakeholder.

Reply:
I'm getting the impression you are right, it seems wg have stakeholders which are sigs, sigs themselves don't have stakeholders. Seems strange to me, I would think other sigs/groups/orgs may have a stake in the sig :man_shrugging: